lms/envs/test.py

```diff
@@ -591,3 +591,92 @@
 LOGISTRATION_API_RATELIMIT = '5/m'
 
 REGISTRATION_VALIDATION_RATELIMIT = '5/minute'
+
+############### ADG #####################
```

Comment:
Let's please update the core change 12 it is not reflecting all of these changes.
```
Added All languages list in lms/envs/test.py and cms/envs/test.py. (I have added a direct commit to koa-master branch and cms/envs/test.py was updated in that commit.)
lms/envs/test.py Added LANGUAGES = [list of all languages]

cms/envs/test.py Added LANGUAGES = [list of all languages]. Added under this commit.

Tests were failing due to updated languages list. We changed languages to make changes for transifex supported langs.

Can override using decorator in tests but this was used at multiple places so added it in test.py.
```

Reply:
These changes were made under different PRs. All other changes made here are listed in core change [8](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes#8-mailchimp-integration-pr-lp-2447).